src/test/scala/com/cognite/sdk/scala/common/LoginTest.scala

```diff
@@ -18,6 +18,5 @@ class LoginTest extends SdkTestSpec {
       new Login(RequestSession[Id]("scala-sdk-test", uri"https://api.cognitedata.com", backend, AuthProvider[Id](auth)))
     val status = login.status()
     assert(status.loggedIn)
-    status.project should not be empty
```

Comment:
we shouldn't migrate login.status to oidc credentials. login.status is for inspecting api-keys while token.inspect is used for oidc tokens.

login.status may work for oidc tokens from time to time but it will not work fully or give 404 (unstable). It is not to be used for oidc tokens.